src/Symfony/Component/VarDumper/Test/UnicodeCharactersCompatibilityTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Test;
+
+trait UnicodeCharactersCompatibilityTrait
```

Comment:
Cause this is a bugfix, it should be a private method in the test, a new trait should be added in 7.1, not here IMO.

Reply:
Ok, so to be sure if I understand your point correctly, I should follow the KISS pattern in my hotfix and let refactoring for later ?

Reply:
Exactly, adding new class as bugfix is not a good idea, refactoring shouldn't be done in bugfix releases till is not really fixing some bug, here it's simple new private method, in tests area used by one test, thus adding it directly would be IMO best choice.